src/main/resources/org/jenkinsci/plugins/scm_filter/GitHubPullRequestSkipTrait/help.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Defines whether a pull request creation or sync should be excluded.
+    Will be so if the PR title has either [ci skip] or [skip ci] (case insensitive).
```

Comment:
I thought we would use [WIP] instead? Where do you find the convention of ci skip or skip ci being used?

Reply:
Many Jenkins plugin uses the keyword [CI SKIP] for skipping CI builds, although used in the context of individual commits rather than branches/pull requests.

Here are some examples:
* [jenkins-ci-skip-plugin](https://github.com/banyan/jenkins-ci-skip-plugin)
* [bitbucket-scm-trait-commit-skip](https://plugins.jenkins.io/bitbucket-scm-trait-commit-skip)
* [github-scm-trait-commit-skip](https://plugins.jenkins.io/github-scm-trait-commit-skip)

I think it stemmed from the built-in feature in Travis to skip building certain commits: 
https://docs.travis-ci.com/user/customizing-the-build/#Skipping-a-build

Reply:
Interesting, I think those keywords are for different level, ci skip (or ci-skip, skip-ci etc.) are for commit, and WIP is usually for PR level. So let's support both if we can (and all the different format/syntax/order of ci-skip

It appears that gitlab plugin for jenkins supports both,
https://github.com/jenkinsci/gitlab-plugin

And github version doesn't support either.
https://github.com/jenkinsci/github-plugin

Additionally, maybe its more appropriate that we fork the official github-plugin and add in the support instead?

Reply:
Ok, I will change it later.

Actually, what we use is based on [GitHub Branch Source plugin](https://plugins.jenkins.io/github-branch-source), a different plugin that is mostly unrelated to github-plugin. This is the plugin that allows us to add the entire Freedom! organization to be configured as a job. This plugin implements the system for branch and repository filtering.

This plugin is actually used in developing this. They made their plugin modular so that people can create scanning/checkout/filtering behaviors by creating a new plugin that uses their API. This way, people can only install those behaviors that they need.